src/Sylius/Behat/spec/Context/Domain/ProductContextSpec.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Behat\Context\Domain;
+
+use Behat\Behat\Context\Context;
+use Doctrine\DBAL\DBALException;
+use PhpSpec\Exception\Example\FailureException;
+use PhpSpec\Exception\Example\NotEqualException;
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Sylius\Behat\Context\Domain\ProductContext;
+use Sylius\Component\Core\Model\ProductVariantInterface;
+use Sylius\Component\Core\Repository\ProductVariantRepositoryInterface;
+use Sylius\Component\Core\Test\Services\SharedStorageInterface;
+
+/**
+ * @mixin ProductContext
```

Comment:
Very good feature my friend, it is, use it and, live in wealth, you will.  Yeesssssss. 
![yoda](https://cloud.githubusercontent.com/assets/6213903/13495244/d7bdd6fe-e149-11e5-99fc-d1e320e52851.jpg)